src/react/script.js

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs';
+import React from 'react';
+
+// The script bellow was minified using terser: https://xem.github.io/terser-online/
+// You may find the original script below in case you need to modify it:
+/*
+(function () {
+if (!Intl.PluralRules || !Intl.RelativeTimeFormat) {
+    var currentScript = document.currentScript;
+    var polyfillScript = document.createElement('script');
+
+    polyfillScript.src = '${polyfillChunkPublicPath}';
+
+    currentScript.parentNode.insertBefore(polyfillScript, currentScript.nextSibling);
+})();
+*/
+const buildScript = (polyfillChunkPublicPath) => `(function(){if(!Intl.PluralRules||!Intl.RelativeTimeFormat){var e=document.currentScript,t=document.createElement("script");t.src="${polyfillChunkPublicPath}",e.parentNode.insertBefore(t,e.nextSibling)}})();`;
```

Comment:
why not use the `raw-loader`, and put this function in another file?

Reply:
Because I would need to setup webpack or rollup into this package, making this more complicated for something simple.